DomainContent/AvatarStore/CheckoutZone/checkoutZone.js

```diff
@@ -181,7 +183,13 @@
             jointName: MyAvatar.jointNames[entityProperties.parentJointIndex],
             demoEntityID: entityID
         };
-  
+
+        var mousePress = (function(id, event) {
+            Overlays.editOverlay(id, {position: scannerPosition});
```

Comment:
This could move any overlay on mousepress, please check if it is an overlay that is part of the checkout